qiita_core/support_files/config_test.txt

```diff
@@ -122,13 +122,23 @@ EBI_CENTER_NAME = CCME-COLORADO
 # study aliases
 EBI_ORGANIZATION_PREFIX = qiime
 
+
+# ----------------------------- IPython settings -----------------------------
 [ipython]
 # context is expected to be a comma separated list
 context = qiita-general
 default = qiita-general
 
+# ----------------------------- VAMPS settings -----------------------------
 [vamps]
 # general info to submit to vamps
 USER = user
 PASSWORD = password
 URL = https://vamps.mbl.edu/mobe_workshop/getfile.php
+
+# ----------------------------- Portal settings -----------------------------
+[portal]
+# Full path to portal styling config file
+PORTAL_FP =
+# Full path to portal custom CSS styling file
+CSS_FP =
```

Comment:
The semantic separation between portal specific attributes makes more sense to me than value types. If not, we might as well just have everything in a single file. This is not a blocking comment though.